src/kibana/components/doc_viewer/doc_viewer.less

```diff
@@ -21,4 +21,24 @@ doc-viewer .doc-viewer {
   .content {
     margin-top: @padding-base-vertical;
   }
-}
\ No newline at end of file
+
+  .null {
+    color: gray;
+  }
+
+  .string {
+    color: red;
+  }
+
+  .number {
+    color: blue;
+  }
+
+  .boolean {
+    color: green;
+  }
+
+  .key {
+    color: darkblue;
+  }
+}
```

Comment:
@rashidkpc Just so I understand - should I create some new LESS variables, eg. json-string, json-number in theme/_variables.less and refer to them here? Or re-use existing variables like @state-success-text etc?